README.md

```diff
@@ -1,56 +1,76 @@
-
-To enable the People API:
-1. Visit the [API Library](https://console.cloud.google.com/apis/library/people.googleapis.com) and click "Enable". Make sure you have the right project selected in the top menu.
-
-After the API is enabled, you need to create credentials for the module.
-
-2. From the [Overview](https://console.cloud.google.com/apis/api/people.googleapis.com/overview) of the People API, select "Credentials" on the left.
-3. Click the button "Create credentials" and choose "OAuth client ID"
-4. Select "Other" and give the client a recognizable name, like "MMM-GoogleBirthdaysProvider"
-5. Click "Create"
-
-You should be redirected to the credentials overview
-
-6. Download the newly created credentials and set the file aside for now, you'll need it in a moment.
-
 ## Installation
 
-1. In your `modules/` directory, `git clone https://github.com/PalatinCoder/MMM-GoogleBirthdaysProvider.git`
-2. `cd` in the new `MMM-GoogleBirthdaysProvider` directory
+1. In your `modules/` directory, `git clone https://github.com/ulrichwisser/MMM-CardDavBirthdaysProvider.git`
+2. `cd` in the new `MMM-CardDavBirthdaysProvider` directory
 3. `npm install --production` (note the production flag, so you don't get all the dev dependencies - you don't need them if you just want to use the module)
-4. Place the JSON file with the credentials (from step 6 above) in `google-api-credentials/credentials.json`
-5. Run `npm run token:generate` and follow the instructions. This will authorize your instance of the module against your GCP project and store an authorization token.
 6. Add the module to your `config.js`:
-   ```
-   {
-       module: "MMM-GoogleBirthdaysProvider",
-       config: {}
-   },
-   ```
-   Note: Don't give the module a position, as it doesn't render any DOM at all
-
-7. Add `http://localhost:8080/mmm-googlebirthdaysprovider` to your calendar URLs, something like this:
+```
+{
+	module: "MMM-CardDavBirthdaysProvider",
+	position: "top_left",
+	config: {
+		authMethod: 'Basic',
+		serverUrl: 'https://secure.carddav.example/',
+		credentials: {
+			username: 'username',
+			password: 'password',
+		},
+	},
+},
+```
+
+Note: the module does not display anything. So you can chose any position of your liking. 
+
+MMM-CardDavBirthdaysProvider uses (tsdav)[https://www.npmjs.com/package/tsdav] from npm.io to retrieve carddav data. Please check the tsdav documentation for details on the credentials.
+
+config should have three attributes "authMethod", "serverURL" and "credentials".
+
+E.g
+```
+config : {
+	authMethod: 'Basic',
+	serverUrl: 'https://secure.carddav.example/',
+	credentials: {
+		username: 'username',
+		password: 'password',
+	},
+},
+```
+or
+```
+config : {
+	authMethod: 'Oauth',	
+	serverUrl: 'https://apidata.googleusercontent.com/caldav/v2/',
+	credentials: {
+		tokenUrl: 'https://accounts.google.com/o/oauth2/token',
+		username: 'YOUR_EMAIL_ADDRESS',
+		refreshToken: 'YOUR_REFRESH_TOKEN_WITH_CALDAV_PERMISSION',
```

Comment:
Can you please share instructions or a script for generating this token?

Reply:
This is actually a pull request. And an old one at that. Totally the wrong place to ask for help.

That said, you need to follow the install instructions on https://github.com/PalatinCoder/MMM-GoogleBirthdaysProvider.
Good luck!

Reply:
Yes, I am aware :) I have been following your updates to your fork and I noticed you are moving to CalDAV API and I am trying to get it working as well. Since you don't have the issues open on your fork, I figured I'll ask here. Let me know if you have a better suggestion for asking questions/ reporting issues for your fork.

Reply:
I have enabled issues and discussions on my repo, please feel free to take the question there.